packages/platform-sdk-generator/src/isIgnoredNamespace.ts

```diff
@@ -14,16 +14,38 @@
  * limitations under the License.
  */
 
-export function isIgnoredNamespace(ns?: string): boolean {
+export function isIgnoredNamespace(
+  ns?: string,
+  packagePrefix: string = "foundry",
+): boolean {
+  // Always ignored
   switch (ns) {
     case "Operations":
       return true;
-    case "TargetWorkbench":
-      return true;
-    case "Gaia":
-      return true;
-    case "MapRendering":
-      return true;
+    case "Core":
+      return false;
   }
-  return false;
+
+  const isFoundryIgnored = () => {
+    switch (ns) {
+      case "TargetWorkbench":
+        return true;
+      case "Gaia":
+        return true;
+      case "MapRendering":
+        return true;
+    }
+    return false;
+  };
+
+  if (packagePrefix === "all") {
+    return false;
+  }
```

Comment:
When would we need to publish both foundry and gotham namespaces in the same package?